test-case/check-pause-release-suspend-resume.sh

```diff
@@ -13,6 +13,8 @@ set -e
 ##    have system enter suspend state for 5 secs
 ##    resume from suspend state
 ##    release audio stream from paused state
+##    repeat to all pipelines
```

Comment:
the test description is not clear

line 15: is the stream always paused prior to suspend? Or can this be random/variable?
line 16: can we have multiple streams in parallel? This will be required for our tests, in fact if you see https://github.com/thesofproject/linux/issues/3151 I am precisely having issues with suspend-resume with 2 paused streams.

Reply:
In this PR, we will first implement a single pipeline test of pause-release-suspend-resume, and then we will re-prove a PR to implement multiple pipeline tests.